vector-app/src/debug/java/im/vector/app/features/debug/features/DebugVectorFeatures.kt

```diff
@@ -76,6 +76,15 @@ class DebugVectorFeatures(
     override fun isNewAppLayoutFeatureEnabled(): Boolean = read(DebugFeatureKeys.newAppLayoutEnabled)
             ?: vectorFeatures.isNewAppLayoutFeatureEnabled()
 
+    override fun isQrCodeLoginEnabled() = read(DebugFeatureKeys.qrCodeLoginEnabled)
+            ?: vectorFeatures.isQrCodeLoginEnabled()
+
+    override fun allowQrCodeLoginForAllServers() = read(DebugFeatureKeys.allowQrCodeLoginForAllServers)
```

Comment:
The naming of the `allow` feature seems confusing in my opinion. Could we rename them to `isQrCodeLoginForAllServersAllowed` and `isReciprocateQrCodeLoginAllowed` to align with others?

Reply:
Agreed, done.